examples/ads.amp.html

```diff
@@ -252,6 +252,16 @@ <h2 id="adhese">Adhese</h2>
     <div fallback></div>
   </amp-ad>
 
+  <h2 id="adfox">AdFox</h2>
+
+  <amp-ad width="248" height="408"
+          type="adfox"
+          data-owner-id="208087"
+          data-adfox-params='{"pt": "b","p1": "bsoji","p2": "feil","pct": "a","pfc": "bbhfo","pfb": "cwrtv"}'>
+    <div placeholder></div>
```

Comment:
We now support default placeholder and fallback. No need to add placeholder and fallback here